src/QirRuntime/lib/QIR/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ compile_from_qir(bridge-qis ${bridge_qis_target})
 set(qis_sup_source_files
   "intrinsics.cpp"
   "intrinsicsMath.cpp"
+  intrinsicsOut.cpp

```

Comment:
Is it significant that this isn't in quotes like the other files?

Reply:
I suspect the quotes started when (or in a project where) the file names contained the chars (like spaces) that were not ok without the quotes. My added file name does not contain such chars so the quotes would be redundant (I may be wrong since I'm new to `cmake`, but I suspect that this part of functionality is the same as for `make` which I dealt with). 
So I skipped the quotes. Somebody please correct me if I'm wrong.

Reply:
I believe you are right. Because I'm also new to cmake, a considerable portion of the initial coding of the CMakeLists.txt files happened by copy-paste from various examples so there are inconsistencies in formatting wrt optional things like quotes. Please feel free to standardize the files you touch (or add to the list of cleanups in https://github.com/microsoft/qsharp-runtime/issues/482)